polaris-react/src/components/ActionList/components/Section/Section.tsx

```diff
@@ -70,6 +70,19 @@ export function Section({
     </Box>
   ) : null;
 
+  const helpTextMarkup = section.helpText ? (
+    <Box
+      paddingBlockStart={section.title ? '0' : '2'}
+      paddingInlineStart="4"
+      paddingBlockEnd="2"
+      paddingInlineEnd="4"
+    >
+      <Text as="span" color="subdued" breakWord>
+        {section.helpText}
```

Comment:
I'm not super familiar with this component but did we have access to this previously but not render it?

Reply:
It was accessible in `section.item[n].helpText` but not directly in `section`. It's going to help with things like [email](https://screenshot.click/05-38-48891-71953.png) under the title or other subtitles.

Reply:
If we are making section `title` (or `header`) a ReactNode then we probably don't need the help text unless we're doing it for stylistic reasons, but the passed in component it can match the styling if it choses to as well.